util/make_new_dif/ip.py

```diff
@@ -81,15 +85,18 @@ class Ip:
         irqs (List[Irq]): List of Irq objs constructed from HJSON data.
 
     """
+
     def __init__(self, name_snake: str, name_long_lower: str,
-                 templated_modules: List[str], ipgen_modules: List[str]) -> None:
+                 templated_modules: List[str], ipgen_modules: List[str],
+                 top_reggen_modules: List[str]) -> None:
```

Comment:
optional nit: If you go with `reggen_top`, that will match the parameter in the hjson.

Reply:
fixed.